### ISSUE TITLE
Failed to increase DynamoDB throughput (Table in use)

### ISSUE THREAD

**Issue by AlexanderC** [MitocGroup/deep-framework#311]
deep-db fails to increase DynamoDB throughput in case there is other increase operation in progress.

Solution: Please catch the `Table IOPS are currently being updated.` exception and set a timeout of 200 ms and retry the operation WITOUT increasing the throughput.

Here's the trace:
```
Failed on increase throughput for table 'DeepProdBook2f11c90b': ResourceInUseException: Attempt to change a resource which is still in use: Table IOPS are currently being updated. Table: DeepProdBook2f11c90b
```

**Comment by CCristi** [comment]
Fixed in deep-db@1.8.1

**Comment by CCristi** [comment]
This error means that dynamodb throughput is increasing too fast. Increase table default throughput in order avoid such kind of errors
